src/client/view/mostUsedWords.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { SlidingSearchBar } from './searchBar';
+
+const WordInfo = ({ wordInfo }) => (
+  <tr>
+    <td className='right aligned column'>{wordInfo.count}</td>
+    <td className='left aligned column'>{wordInfo.word}</td>
+    <td className='right aligned column'>
+      <div className='ui checkbox'>
+        <input type='checkbox'  defaultChecked='true'/>
+      </div>
+    </td>
+  </tr>
+);
+
+const Words = React.createClass({
+  componentDidMount() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  componentDidUpdate() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  render() {
+    return (
+      <div style={{ 'height': '300px', 'overflow-y': 'scroll' }}>
+        <table className='ui very basic celled table'>
+          <tbody>
+          { this.props.words.map((wordInfo) => <WordInfo wordInfo={wordInfo}></WordInfo>) }
+          </tbody>
+        </table>
+      </div>
+    );
+  },
+});
+
+const ToggleAllWords = () => (
+  <button className='ui button'>Hide all</button>
+);
+
+export const MostUsedWords = React.createClass({
+  getInitialState() {
+    return { 'search': '' };
+  },
+

```

Comment:
Remove initial state and pass as prop... change to stateless functional React Class